python/src/trezorlib/messages.py

```
@@ -7222,6 +7225,20 @@ def __init__(
         self.destination_tag = destination_tag
 
 
+class RippleAccountDelete(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = None
+    FIELDS = {
+        1: protobuf.Field("destination", "string", repeated=False, required=True),
+    }
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)